qa/automation/CareerPortal.Tests.Scenarios/Savostin/buttons.feature

```diff
@@ -0,0 +1,52 @@
+Feature: 1.0.0 buttons lead to correct pages
+Background: 
+	Given "Career Quantori" page is opened
```

Comment:
I would recommend to add URL (https://career.quantori.com/) instead of just the name of the website in order to avoid any misunderstanding in the future. :)